include/nanobind/nb_misc.h

```diff
@@ -39,4 +39,14 @@ inline void set_implicit_cast_warnings(bool value) noexcept {
     detail::set_implicit_cast_warnings(value);
 }
 
+/// \ingroup python_builtins
+/// Return a dictionary representing the global variables in the current execution frame,
```

Comment:
That whole comment is now out of sync and can simply be deleted. (FWIW nanobind also don't use the `\ingroup` syntax since the API docs are separately built using Sphinx.